src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerManager.cs

```diff
@@ -235,6 +235,7 @@ private void ClearCompilationScopeMap(AnalyzerAndOptions analyzerAndOptions, Ana
                 });
 
                 // Subscribe for exceptions from lazily evaluated localizable strings in the descriptors.
+                // PREVIEW: when are these handle unsubscribe from description?
```

Comment:
`REVIEW` not `PREVIEW`.  Also, I would word this as "When do these unsubscribe from description?"